frontend/public/assets/index-banner.svg

```diff
@@ -0,0 +1,51 @@
+<svg width="379" height="254" viewBox="0 0 379 254" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe rename it from index-banner to something more appropriate aha, index-banner doesn't tell me much :P

Reply:
oh fuck I forgot this one. I've no idea what to name it. help me here.